src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -406,7 +406,7 @@ public void setBidderAddressBookFilePath(Path bidderAddressBookFilePath) {
     }
 
     @Override
-    public void updateFilteredBidderList(Predicate<Bidder> predicate) {
+    public void updateFilteredBidderList(Predicate<? super Person> predicate) {
```

Comment:
Seems ok but curious what scenario would you need to add in a Person instead of a bidder in the unfiltered bidder list predicate?